docs/api.rst

```diff
@@ -235,6 +237,29 @@ API Reference
    :members: wrap_for_formatter
 
 
+`structlog.types` Module
+------------------------
+
+.. automodule:: structlog.types
+
+.. autoprotocol:: BindableLogger
+
+   Additionally to the methods listed below, bound loggers **must** have a ``__init__`` method with the following signature:
+
+   .. method:: __init__(self, wrapped_logger: WrappedLogger, processors: typing.Iterable[Processor], context: Context) -> None
```

Comment:
```suggestion
   .. method:: __init__(self, wrapped_logger: WrappedLogger, processors: Iterable[Processor], context: Context) -> None
```

Per [PEP 585](https://www.python.org/dev/peps/pep-0585/), `collections.abc.Iterable[Processor]` is probably also acceptable :wink: